README.md

```diff
@@ -412,7 +412,9 @@ group so as to allow access to `/dev/cpu/*/rmsr`.
 ## Security Notes
 
 Krun is not intended to be run on a secure multi-user system, as it uses sudo
-to elevate privileges.
+to elevate privileges. It also uses files with fixed names in /tmp/ which means
+that only one instance of Krun should be run at any one time (running more
+than one leads to undefined behaviour).
```

Comment:
This is true. In the past I've accidentally run >1 krun, and this also tends to upset the performance counter code, since each instance resets the core-cycle counter, it's possible for one instance to observe a lower value after the pexec than before :)

So even if we did fix the filename thing, you still couldn't run >1 instance :)